tasks/upload_image.yml

```diff
@@ -23,6 +23,8 @@
     state: present
     validate_certs: "{{ validate_certs }}"
     filename: "{{ images_path + generator_image_filename }}"
+    properties:
+      hw_vif_multiqueue_enabled: true
```

Comment:
Hello @glm35 

TRex does not use this image property ?
Is it necessary for generator image ?

Reply:
Hello @frmenguy,

In case the traffic generator VM is deployed on a SR-IOV compute node (with SR-IOV or PCI-passthrough network interfaces for traffic generation), this parameter has no effect (it is both useless and harmless).

But in case the traffic generator VM is deployed on a DPDK compute node, this parameter is super useful to get decent perf: it allows TRex to parallelize the generation of outgoing packets.

Reply:
Ok, good for me. Thanks for this patch.